generate-geodatabase-replica-from-feature-service/src/main/java/com/esri/arcgismaps/sample/generategeodatabasereplicafromfeatureservice/MainActivity.kt

```diff
@@ -114,6 +129,8 @@ class MainActivity : AppCompatActivity() {
 
         // set the button's onClickListener
         generateButton.setOnClickListener {
+            // close the current geodatabase, if a replica was already generated
+            geodatabase?.close()
```

Comment:
Note, you can only close a gdb if it's not in use anymore. Is the map that displays data of this gdb still being displayed at this point here? If so, it's not safe to call close here.